epochStart/shardchain/trigger.go

```diff
@@ -454,26 +454,32 @@ func (t *trigger) updateTriggerFromMeta() {
 		// save all final-valid epoch start blocks
 		if canActivateEpochStart {
 			t.mapFinalizedEpochs[currMetaInfo.hdr.Epoch] = struct{}{}
-			epochStartIdentifier := core.EpochStartIdentifier(currMetaInfo.hdr.Epoch)
+			t.saveEpochStartMeta(currMetaInfo.hdr)
+		}
+	}
+}
+
+func (t *trigger) saveEpochStartMeta(metaHdr *block.MetaBlock) {
+	if check.IfNil(metaHdr) {
+		return
+	}
 
-			log.Debug("saving epoch start meta with", "key", epochStartIdentifier)
+	epochStartIdentifier := core.EpochStartIdentifier(metaHdr.Epoch)
```

Comment:
You still can add this log: log.Debug("saving epoch start meta with", "key", epochStartIdentifier)

Reply:
It was only for my simple purpose. No need